lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -226,8 +228,8 @@ class ReportUIFeatures {
     this.headerSticky.style.transform = `translateY(${heightDiff * scrollPct * -1}px)`;
     this.headerBackground.style.transform = `translateY(${scrollPct * this.headerOverlap}px)`;
     this.lighthouseIcon.style.transform =
-      `translate3d(var(--report-width-half),` +
-      ` calc(-100% - ${scrollPct * this.headerOverlap * -1}px), 0) scale(${1 - scrollPct})`;
+      `translate3d(0,` +
```

Comment:
you might have to explain this one to me :)

Reply:
Since the `lh-lighthouse` element is inside the header container now it doesn't need to be translated on the x axis since it is positioned absolutely to the bottom right corner.

Reply:
and the `100%` is gone because of the `right: 0` we're apply all the time now?

Reply:
Yup, it had -100% on in the `.lh-lighthouse` class to position it before, so the new anchoring doesn't need it.